creme/tree/hoeffding_adaptive_tree_classifier.py

```diff
@@ -0,0 +1,238 @@
+from creme.tree import HoeffdingTreeClassifier
+from creme.utils.skmultiflow_utils import add_dict_values
+from creme.utils.math import softmax
+
+from ._nodes import InactiveLeaf
+from ._nodes import InactiveLearningNodeMC
+from ._nodes import FoundNode
+from ._nodes import SplitNode
+from ._nodes import LearningNode
+from ._nodes import AdaLearningNodeClassifier
+from ._nodes import AdaSplitNodeClassifier
+
+
+class HoeffdingAdaptiveTreeClassifier(HoeffdingTreeClassifier):
+    """Hoeffding Adaptive Tree classifier.
+
+    Parameters
+    ----------
+    grace_period
+        Number of instances a leaf should observe between split attempts.
+    split_criterion
+        | Split criterion to use.
```

Comment:
I am not completely sure if this formating will work in markdown. Just mentioning to keep it in mind as it is also used in other places.

Reply:
I also share the same doubt. I leveraged the previous version of the docstring and made minor changes.

Reply:
> Besides that, I have the feeling the file history only goes until the first commit in `the-merge`, at least on the web interface. Does it sound right?

There are files that were correctly moved and the tracking goes back in the sub-tree history:

![image](https://user-images.githubusercontent.com/17923265/96358418-807c5b00-1163-11eb-8070-efc1e3cd7409.png)

However, there are other files that are identified as new, and tracking history is lost.

![image](https://user-images.githubusercontent.com/17923265/96358446-bae5f800-1163-11eb-900b-c41708f53ac8.png)

Honestly, I have no idea if this is fixable (more importantly in an easy way). I will look into it tomorrow.